recipes/opencl-icd-loader/all/conanfile.py

```diff
@@ -1,31 +1,35 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile, conan_version
```

Comment:
Sounds like you're supposed to remove the `conan_version` here.

Reply:
Thank you for catching it :)